utils/src/logger.test.ts

```diff
@@ -0,0 +1,131 @@
+import assert from 'node:assert/strict';
+import { describe, it } from 'node:test';
+import { execPath, platform } from 'node:process';
+import { spawnPromisified } from './spawn-promisified.ts';
+
+const skipWindows = platform === 'win32';
+
+describe('Logger', { skip: skipWindows, concurrency: true }, () => {
+	it('should log info messages', async (t) => {
+		const { code, stdout } = await spawnPromisified(
+			execPath,
+			[
+				'--no-warnings',
+				'--experimental-strip-types',
+				'-e',
+				`
+				import { info } from './logger.ts';
+				info('This is an info message');
+			`,
```

Comment:
Can we add `dedent` as a dev dep for this? I feel like it'll make it look nicer

Reply:
Why not but it's test and we don't look at how indentation is. SO not really useful

Reply:
I just think it'll look nicer, since, if this errors, the logs will have `				` in them, rather than just starting right at the import

Reply:
humm make sense